sympy/functions/combinatorial/factorials.py

```diff
@@ -852,7 +852,7 @@ def _eval_expand_func(self, **hints):
             return binomial(*self.args)
 
     def _eval_rewrite_as_factorial(self, n, k):
-        return factorial(n)/(factorial(k)*factorial(n - k))
+        return "binomial(n,k)"
```

Comment:
This rewrite should not be changed. Only combsimp should be changed to rewrite this combination of factorials as a binomial.